lspd/docker-compose.yml

```
@@ -9,6 +9,12 @@ services:
       POSTGRES_USER: lipauser
       POSTGRES_PASSWORD: lipapassword
       POSTGRES_DB: lipadb
+    healthcheck:
+      test: ["CMD", "psql", "-U", "lipauser", "lipadb"]
```

I think this can be made simpler and more generalised.
```suggestion
      test: ["CMD-SHELL", "pg_isready"]
```